plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -155,7 +155,7 @@ export const TechDocsReaderPageHeader = (
       type="Documentation"
       typeLink={docsRootLink}
       title={title || skeleton}
-      subtitle={subtitle || skeleton}
+      subtitle={subtitle !== '' && (subtitle || skeleton)}
```

Comment:
Hey @omerfarukdogan, thanks for the PR!

If subtitle is `''`, `subtitle !== '' && (subtitle || skeleton)` will evaluate to `false` which will behave correct now, but I feel like it could cause problems in the future.

Don't you think it would be safest to make sure we only pass a string, `undefined` or even `''` in the `subtitle` prop?

Reply:
Hello @agentbellnorm,

Would this be better?

`subtitle={subtitle === '' ? undefined : (subtitle || skeleton)}`

Reply:
@omerfarukdogan Yeah, it seems like that will not evaluate to a boolean!

Reply:
Thanks @agentbellnorm, I updated the pull request.